src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -600,6 +603,16 @@ def launch_fluent(
 
     del start_container
 
+    # Check if user set PYFLUENT_ADDITIONAL_ARGUMENTS var
```

Comment:
I suppose that I don't agree with this behaviour. The arguments I explicitly pass to the launch_fluent call should always take precedence over the configured environment.

Reply:
In my understanding, the purpose of this environment variable was specifically to override whatever is defined in scripts. 

However, I can see now that this could cause some unexpected/unintuitive behavior: defining this env var this way will also cause Python interpreter calls to `launch_fluent` with `additional_arguments` to be ignored. This is not necessarily an issue, but certainly an undesirable side-effect.

In this case, I think it makes the most sense to revert to the behavior that had been initially suggested by @schrummy14, where the actual arguments specified for `launch_fluent` take precedence over what is defined in the env var. Thoughts @schrummy14 @mkundu1 ?